geomstats/geometry/special_euclidean.py

```diff
@@ -767,80 +767,3 @@ def exponential_matrix(self, rot_vec):
         assert exponential_mat.ndim == 3
```

Comment:
You didn't move the private function `exponential_matrix`, aren't you using it?

Reply:
I'm not using it, unless `self.exp` calls it. Was I supposed to move it?

Reply:
The exponential barycenter in SE(3) needs it, see other comment.